library.properties

```diff
@@ -1,9 +1,9 @@
 name=losant-mqtt-arduino
-version=1.3.0
+version=1.4.0
```

Comment:
does this deserve a major bump cause of the json changes?

Reply:
I agree with making this a major bump since the top-level API is no longer compatible with v1.x.